RecoLocalCalo/Configuration/python/RecoLocalCalo_Commissioning_cff.py

```diff
@@ -26,11 +26,13 @@
 #
 # sequence CaloLocalReco and CaloGlobalReco
 #
-calolocalreco = cms.Sequence(ecalLocalRecoSequence+hcalLocalRecoSequence)
+calolocalrecoTask = cms.Task(ecalLocalRecoTask,hcalLocalRecoTask)
+calolocalreco = cms.Sequence(calolocalrecoTask)
 caloglobalreco = cms.Sequence(hcalGlobalRecoSequence)
```

Comment:
Why not also migrating this sequence?

Reply:
Thank you for your every comments. 
I kept this sequence that have one module(hbhereco).
In this PR, I modified the sequences that has more than one modules.

In  cmssw/RecoLocalCalo/Configuration/python/hcalGlobalReco_cff.py 
`from RecoLocalCalo.HcalRecProducers.HBHEIsolatedNoiseReflagger_cfi import *`
`hcalGlobalRecoSequence = cms.Sequence(hbhereco)`

Reply:
Indeed, this sequence is never used, it only includes ```hcalGlobalRecoSequence``` from ```RecoLocalCalo/Configuration/python/hcalGlobalReco_cff.py``` (which in its turn is only used for the ```globalreco_tracking``` reconstruction sequence) and it can be misleading here: I would suggest to just remove it